plugins/main/ipvlan/ipvlan.go

```diff
@@ -146,6 +156,10 @@ func createIpvlan(conf *NetConf, ifName string, netns ns.NetNS) (*current.Interf
 		ipvlan.Mac = contIpvlan.Attrs().HardwareAddr.String()
 		ipvlan.Sandbox = netns.Path()
 
+		// TODO: Bug?  ipvlan's ParentIndex isn't set
```

Comment:
I think here you want MasterIndex, not ParentIndex. Bridge ports, macvlans, ipvlans, and team interfaces will have MasterIndex == to their master interface (IFLA_MASTER), but others like veth will have their ParentIndex set to their peer (IFLA_LINK).